playbooks/common/openshift-node/config.yml

```diff
@@ -27,10 +27,12 @@
     stat:
       path: "{{ item }}"
     with_items:
-    - "/etc/openshift/node/node.key"
-    - "/etc/openshift/node/node.kubeconfig"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.crt"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.key"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.kubeconfig"
```

Comment:
If you're going to be building your own node config yourself anyway, I agree it's better to use the building block commands to create the certs, especially now that create-api-client-config lets you specify a cleaner basename for the generated cert/key/kubeconfig files if you want. Calling create-node-config and counting on particular cert/key filenames probably isn't a good idea.